### DISCUSSION TITLE
CR-6 touch display

### DISCUSSION THREAD

**Discussion by tnagy294** [Jyers/Marlin#181] [Q&A] [answered]
I see there some lines of code for using CR-6 touch display, has someone tried it in real? I have one display original for CR-6 and would like to try out but not sure how to do it... I can set some necessary changes in the code, can compile but not sure which files to use...could someone help me a bit?
Many thanks.

**Comment by Jyers** [comment, marked as answer]
You can use the CR-6 display, but it will not be using my new menu, it would be using whatever menu is already set up in marlin for the CR-6. To enable it you would have to comment out the Creality dwin ExtUI option and uncomment the CR-6 display option.

**Reply by tnagy294** [reply]
Thanks, but no success :-( . Unzipped to a new directory, made a change in configuration.h (that in Marlin directory) compiled but nothing...display shows only the startup logo with an empty progress bar... What about the wiring? I use TX2 and RX2 on lcd to TX and RX on the printer side...
BTW your compiled version for E3 V2 incl. the menus is just amazing...

**Reply by Jyers** [reply]
I'm not sure exactly what could be causing that as I've never worked with a CR-6 display. Maybe look around online to see what could be causing the issue.

**Reply by tnagy294** [reply]
OK, thanks. I have no experiences with serial displays, not sure about the wiring for cr-6 lcd. E3V2 display works great, changed the icons to have more colors (uploaded new 9.ico file)...would like to try some touch displays but till now no luck...not giving up :-). I have been looking for some projects with touch lcd for ender 3 v2 but haven't found usefull solutions for me...maybe later with more people having more experiences...I also would like to get more knowledge about creating menus for e3v2 printer with these new displays...how it works, how to write, what's important to think of and so...

**Comment by tnagy294** [comment]
There's a new FW for Ender 3 using touch display...I own one for CR-6 SE and tried to flash and it works perfectly, but onlu a compiled FW is downloadable, no source (maybe later). So it is possible to use CR-6 SE touch display for Ender 3 (I have Ender 3 V2)...it's a pleasure to use touch displays...just FYI
